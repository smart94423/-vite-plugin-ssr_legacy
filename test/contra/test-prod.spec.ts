import { beforeAll } from 'vitest'
import { build } from 'vite'
import { test } from './test'

beforeAll(async () => {
  await buildApp()
}, 40 * 1000)

test()

async function buildApp() {
  const inlineConfig = {
    logLevel: 'warn' as const,
    root: __dirname,
    configFile: __dirname + '/vite.config.js'
  }
  await build(inlineConfig)
  await build({
    build: { ssr: true },
    ...inlineConfig
  })
}
