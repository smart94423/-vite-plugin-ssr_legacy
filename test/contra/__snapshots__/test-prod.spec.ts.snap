// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Contra Simulation > private properties 1`] = `"<script id=\\"vite-plugin-ssr_pageContext\\" type=\\"application/json\\">{\\"abortReason\\":\\"!undefined\\",\\"_urlRewrite\\":null,\\"_urlRedirect\\":\\"!undefined\\",\\"_hasAdditionalPageContextInit\\":\\"!undefined\\",\\"_pageId\\":\\"/pages/index\\"}</script>"`;
